measure/src/test/kotlin/com/alliander/open/measure/MeasureTest.kt

```diff
@@ -8,36 +8,21 @@ import com.alliander.open.measure.Energy.Companion.joule
 import com.alliander.open.measure.Energy.Companion.kiloJoule
 import com.alliander.open.measure.Energy.Companion.kiloWattHour
 import com.alliander.open.measure.Energy.Companion.megaJoule
-import com.alliander.open.measure.Energy.Companion.megaWattHour
 import com.alliander.open.measure.Power.Companion.kiloWatt
 import com.alliander.open.measure.Power.Companion.megaWatt
-import com.alliander.open.measure.Power.Companion.watt
 import com.alliander.open.measure.Time.Companion.hours
 import com.alliander.open.measure.Time.Companion.minutes
 import com.alliander.open.measure.Time.Companion.seconds
-import com.alliander.open.measure.extension.sum
 import io.kotest.core.spec.style.StringSpec
 import io.kotest.data.headers
 import io.kotest.data.row
 import io.kotest.data.table
 import io.kotest.matchers.shouldBe
-import io.kotest.matchers.shouldNotBe
 import io.kotest.property.checkAll
 import java.math.BigDecimal
 import java.math.RoundingMode.UP
 
 class MeasureTest : StringSpec({
-    "Joule, Watt and seconds are different units" {
```

Comment:
Moved this to a more elaborate test in the new test file: The units have the correct measure